src/main/resources/fabric.mod.json

```diff
@@ -30,7 +30,7 @@
   "depends": {
     "fabricloader": ">=0.14.21",
     "java": ">=17",
-    "minecraft": ">=1.20",
+    "minecraft": ">=1.20.2",
```

Comment:
the minecraft dependency was `1.20` for the `1.20`, `1.20.1` and `1.20.2` version so we may want to leave it to `1.20` for all `1.20.*` versions?